webauthn-authenticator-rs/src/MacAuthn/README.md

```diff
@@ -0,0 +1,3 @@
+# MacAuthn
+
+A description of this package.
```

Comment:
I know it requires a fully bundled app with entitlements, can you please note the specifics since you've done it previously? It'd save some research for future-folks.